cmake/GitUtils.cmake

```diff
@@ -137,20 +162,29 @@ function(git_clone)
                 COMMAND             ${GIT_EXECUTABLE} fetch --all --tags --prune
                 COMMAND             ${GIT_EXECUTABLE} checkout tags/${PARGS_GIT_TAG} -b tag_${PARGS_GIT_TAG}
                 WORKING_DIRECTORY   ${${SOURCE_DIR}}
+				RESULT_VARIABLE 	git_result
                 OUTPUT_VARIABLE     git_output)
     elseif(PARGS_GIT_BRANCH OR PARGS_GIT_COMMIT)
         execute_process(
                 COMMAND             ${GIT_EXECUTABLE} checkout ${PARGS_GIT_BRANCH} ${PARGS_GIT_COMMIT}
                 WORKING_DIRECTORY   ${${SOURCE_DIR}}
+				RESULT_VARIABLE 	git_result
                 OUTPUT_VARIABLE     git_output)
     else()
         message(STATUS "no tag specified, defaulting to master")
         execute_process(
                 COMMAND             ${GIT_EXECUTABLE} checkout master
                 WORKING_DIRECTORY   ${${SOURCE_DIR}}
+				RESULT_VARIABLE 	git_result
```

Comment:
Wrong indentation

Reply:
Come on.
this is not a python script.

Reply:
I know I know. But it's important to have consistent formatting, wouldn't you agree?